src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -278,6 +278,8 @@ private String getInputArguments() {
                 new ArrayList<Artifact>(Arrays.asList(getPdfReportArtifact(), getTapArtifact(),
                         getJunitArtifact(), getStmResultsArtifact(), getCoberturaArtifact(),
                         getModelCoverageArtifact()));
+        
+        inputArgsList.add("'Test'");
 
         for (Artifact artifact : artifactList) {
             artifact.addFilePathArgTo(args);

```

Comment:
Should we be expanding environment variables in the field values before they we write them to the script file?

I think we were expanding env variables before, so it would be a change of behavior if we didn't.

Reply:
I don't think we were expanding test artifact values before. We were expanding it for Command text box and MATLAB root input text box.  Matrix build maintains individual workspace for each axis that we add hence artifacts will always be unique in each workspace. expansion in artifact text would just allow substituting Env names for freestyle or assigning the same name as of axis in Matrix (Which i feel is not strong use case). However, we could discuss this if we should support expansion for test artifacts.

Reply:
On line 69 of `MatlabRunTestsStepExecution` we used to have:

`matlabFunctionName+ "("+envVars.expand(getCommand()+")")`

And `getCommand()` used to return something like `addpath(...); runMatlabTests('PDFTestReport', 'SOME_FILE_PATH', ...)` right?

Which means if the user specified `SOME_FILE_PATH` as a Jenkins variable (e.g. `${MY_FILE_PATH}`), I think we would have expanded it.

Not sure this is a significant use-case but it seems like it would be a change of behavior if we don't expand them now.

Reply:
Done.